src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyFileMetadata.scala

```diff
@@ -15,13 +15,18 @@
  */
 package nl.knaw.dans.easy.stage.fileitem
 
+import java.io.File
+
 import scala.util.Try
 
 object EasyFileMetadata {
   def apply(s: FileItemSettings): Try[String] = Try {
+      val parentPath = s.pathInDataset.get.getParentFile
```

Comment:
The point I wanted to make here was that you seem to only have the `Try` here to catch exceptions that originate from `Option.get` calls in several places throughout this function, right? That seems unnecessary to me. If you use the appropriate higher order functions, you can stay within the `Option` type that you already have. Then your return type becomes `Option[String]`.